app/src/main/java/com/android/partagix/ui/screens/EditAccount.kt

```diff
@@ -157,20 +181,111 @@ fun EditAccount(
 
                 Spacer(modifier = Modifier.height(16.dp))
                 Row(
-                    modifier = Modifier.fillMaxWidth().padding(8.dp, 0.dp).testTag("actionButtons"),
+                    modifier =
+                        Modifier.padding(16.dp, 0.dp)
+                            .clickable { contact = !contact }
+                            .testTag("contactInfo"),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.Absolute.Left) {
+                      if (contact) {
+                        Icon(imageVector = Icons.Default.KeyboardArrowUp, contentDescription = null)
+                      } else {
+                        Icon(
+                            imageVector = Icons.Default.KeyboardArrowDown,
+                            contentDescription = null)
+                      }
+                      Text("Contact information ")
+                    }
+
+                if (contact) {
+                  Spacer(modifier = Modifier.height(8.dp))
+                  Text(text = "Favorite contact methods", modifier = Modifier.padding(16.dp, 0.dp))
+                  Spacer(modifier = Modifier.height(4.dp))
+                  Row(
+                      modifier = Modifier.fillMaxWidth().padding(start = 16.dp, end = 32.dp),
+                      horizontalArrangement = Arrangement.Absolute.SpaceEvenly) {
+                        Row(
+                            horizontalArrangement = Arrangement.Absolute.Left,
+                            verticalAlignment = Alignment.CenterVertically) {
+                              Checkbox(
```

Comment:
One another cool thing with those Checkboxes would be to hide the Input Field if the corresponding Checkbox is unchecked

Reply:
For me we still need to be able to enter other contact information, for example my favorite is telegram but I know everybody doesn't have telegram so I also put my phone number

Reply:
OH I see, I misunderstand it. Than it works